### DISCUSSION TITLE
post/improve-service-performance/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#50] [Announcements]
# post/improve-service-performance/

카카오페이 온라인 결제 시스템의 성능 개선 기록을 공유합니다.

https://tech.kakaopay.com/post/improve-service-performance/

**Comment by quddnr153** [comment]
레거시 시스템의 성능개선 과정을 볼 수 있는 아주 좋은 글이네요. 좋은 경험 공유 감사합니다. 잘 읽었습니다.

**Comment by pay-dan-dy** [comment]
성능 개선 과정에서 주의 깊게 살펴야 할 부분들을 소중한 경험을 통해 잘 공유해주셨네요~. 감사합니다!

**Comment by KyleLee4087** [comment]
대고수

**Comment by sh26** [comment]
성능 개선 과정에 대한 단계적인 접근을 보기 좋게 정리해주셔서 감사합니다 !
